Docs/source/install/hpc/frontier.rst

```diff
@@ -244,12 +259,4 @@ Known System Issues
       mkdir diags
       # change striping for new files before you submit the simulation
       #   this is an example, striping 10 MB blocks onto 32 nodes
-      lfs setstripe -S 10M -c 32 diags
-
-   Additionally, other AMReX users reported good performance for plotfile checkpoint/restart when using
-
-   .. code-block:: ini
-
-      amr.plot_nfiles = -1
-      amr.checkpoint_nfiles = -1
-      amrex.async_out_nfiles = 4096  # set to number of GPUs used
+      lfs setstripe -S 16M -c 1 $SURM_SUBMIT_DIR  # or diags only or checkpoints only
```

Comment:
Was this meant to be `SLURM` or is it a variable I'm not familiar with?
```suggestion
      lfs setstripe -S 16M -c 1 $SLURM_SUBMIT_DIR  # or diags only or checkpoints only
```

Reply:
Definitely a typo, thanks for spotting it.